src/FeedbackForm/FeedbackForm.js

```diff
@@ -0,0 +1,268 @@
+import * as React from "react";
+import AddIcon from "@mui/icons-material/Add";
+import Button from "@mui/material/Button";
+import Card from "@mui/material/Card";
+import CardContent from "@mui/material/CardContent";
+import CloseIcon from "@mui/icons-material/Close";
+import Dialog from "@mui/material/Dialog";
+import DialogActions from "@mui/material/DialogActions";
+import DialogContent from "@mui/material/DialogContent";
+import DialogTitle from "@mui/material/DialogTitle";
+import { Fab } from "@mui/material";
+import IconButton from "@mui/material/IconButton";
+import PropTypes from "prop-types";
+import SentimentSatisfiedAltIcon from "@mui/icons-material/SentimentSatisfiedAlt";
+import SentimentVeryDissatisfiedIcon from "@mui/icons-material/SentimentVeryDissatisfied";
+import TextField from "@mui/material/TextField";
+import TextsmsOutlinedIcon from "@mui/icons-material/TextsmsOutlined";
+import Typography from "@mui/material/Typography";
+import { styled } from "@mui/material/styles";
+import { useState } from "react";
+
+// for parent dialog
+const BootstrapDialog = styled(Dialog)(({ theme }) => ({
+  "& .MuiDialogContent-root": {
+    padding: theme.spacing(2)
+  }
+}));
+
+const BootstrapDialogTitle = props => {
```

Comment:
I think this dialog title component should be separate component inside the react-ui library. We already have a ticket open to migrate dialog title to new component so would be great if you could do that as part of this work. We use dialog title in few other apps and at the moment it is duplicated so would be nice to have it as separate component.

https://github.com/IPG-Automotive-UK/react-ui/issues/58